operator-pipeline-images/operatorcert/__init__.py

```diff
@@ -261,14 +261,20 @@ def parse_pr_title(pr_title: str) -> Tuple[str, str]:
     If yes, extract the Bundle name and version.
     """
     # Verify if PR title follows convention- it should contain the bundle name and version.
-    # Any non- empty string without whitespaces makes a valid version.
-    regex = rf"^operator ([a-zA-Z0-9-]+) \(([^\s]+)\)$"
+    regex = rf"^operator ([a-zA-Z0-9-]+) \(([0-9]+\.[0-9]+\.[0-9]+[a-zA-Z0-9-]*)\)$"
     regex_pattern = re.compile(regex)
+    err_msg = f"Pull request title {pr_title} does not follow the regex 'operator <operator_name> (<version>)"
+
+    # Check <version> for prefix 'v' failure case and include additional error context
+    regex_v = rf"^operator ([a-zA-Z0-9-]+) \((v[0-9]+\.[0-9]+\.[0-9]+[a-zA-Z0-9-]*)\)$"
+    regex_pattern_v = re.compile(regex_v)
+    err_msg_v = f" where <version> does not contain prefix 'v'"
+
+    if regex_pattern_v.match(pr_title):
```

Comment:
I read this code several times and I have a hard time understanding it. Please correct me if my understanding is wrong but it seems to me that the `v` prefix is not allowed anymore. Is that what you want to achieve here? If so how about operators that passed the certification and have the `v` prefix: https://github.com/redhat-openshift-ecosystem/certified-operators/pull/2523 ?

Reply:
Thanks for the explanation @jsm84 . I am ok with merging it. Thanks for your contribution!